src/components/buttons/_buttons-mixins.scss

```diff
@@ -32,85 +25,213 @@ $buttonActiveScale: scale(0.985);
   white-space: nowrap;
   font-weight: $fontWeightBold;
   text-transform: uppercase;
-  box-shadow: $componentShadow;
-  transition-property: box-shadow, transform, opacity;
+  box-shadow: none;
+  transition-property: background-color, border-color;
   transition-duration: 0.3s;
+  transition-timing-function: ease-in-out;
 
-  // hack for proper text centering on Safari related to inline-flex issue with <button> tag
-  &::before,
-  &::after {
-    content: '';
-    flex: 1 0 auto;
-  }
-
-  &:hover,
-  &:focus {
-    box-shadow: $elevatedComponentShadow;
+  &__container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
   }
 
-  &:active {
-    box-shadow: $componentShadow;
-    transform: $buttonActiveScale;
+  &--full-width {
+    width: 100%;
   }
 
-  &:active:focus {
-    box-shadow: $buttonActiveBoxShadow;
+  &:focus {
+    outline: none;
   }
 
-  &:active,
+  &:hover,
   &:focus,
-  &:hover {
+  &:active,
+  &:active:focus {
+    box-shadow: none;
     text-decoration: none;
+    background-color: mix($mintPrimaryDark, $mintPrimary, 80%);
+    border-color: mix($mintPrimaryDark, $mintPrimary, 80%);
   }
 
-  &__container {
-    display: flex;
-    align-items: center;
-    justify-content: center;
-  }
-
-  &--full-width {
-    width: 100%;
+  &.sg-button-primary--disabled,
```

Comment:
We have now changes in color of each button on hover, so I need to make sure that for disabled buttons does not happen. Probably there is a better way to do it in SASS, but honestly, I consider it as a waste of time to figure it out now, as I will remove all this code soon. 🤷‍♀️I would prefer to put more attention into the new Button component instead. But if you have some ideas I can, of course, make adjustments here 🙃

Reply:
I'd not put it inside mixin maybe...?